templates/docs/examples/patterns/logo-section/logo-section-dark.html

```diff
@@ -3,15 +3,7 @@
 
 {% block standalone_css %}patterns_logo-section{% endblock %}
 
-{% block style %}
-<style>
-body {
-  background: #262626;
-  color: #FFF;
-}
-</style>
-{% endblock %}
-
+{%set is_dark = True%}
```

Comment:
As you mentioned in the comment, this is tricky one because if images. We can't change images based on the theme, we don't have a solution for it (outside of built in icons). We can't change src based on theme or anything, at least not easily.

So I guess it's ok to just keep this one dark, with switch not working.